src/pages/steps/hooks/useFooterStatus.ts

```diff
@@ -16,14 +16,19 @@ const useFooterStatus = ({ offer, form }) => {
   const router = useRouter();
   const queryClient = useQueryClient();
 
+  const formValues = form.getValues();
   const {
     formState: { dirtyFields },
   } = form;
 
   const isMutating = queryClient.isMutating();
   const offerId = offer?.['@id'];
   const availableFrom = offer?.availableFrom;
-  const isPlaceDirty = dirtyFields.place || dirtyFields.location;
+  const hasPlace =
+    (formValues.location?.municipality &&
+      formValues.location?.streetAndNumber) ||
+    formValues.location.isOnline;
+  const isPlaceDirty = (dirtyFields.place || dirtyFields.location) && hasPlace;
```

Comment:
Won't this cause issues for `place` since hasPlace wil never be true in the case of `place`

Reply:
No it works fine I just misnamed the variable, should be `hasLocation` which is true for all offers